[New] Archlinux install instructions (#1292)
timescale/docs: install/installation-archlinux.md

instructions use the `pacman` package manager.

<highlight type="warning">
If you have already installed PostgreSQL using a method other than the `pacman`
package manager, you could encounter errors following these instructions. It is
safest to remove any existing PostgreSQL installations before you begin. If you
want to keep your current PostgreSQL installation, do not install TimescaleDB
using this method.
[Install from source](/install/latest/self-hosted/installation-source/)
instead.
</highlight>

<procedure>

### Installing self-hosted TimescaleDB on archlinux-based systems
1.  Install TimescaleDB and timescaledb-tune:
    ```bash
    pacman -Syu timescaledb timescaledb-tune
    ```
1.  Initialize the database as the postgres user: 
    ```bash
    sudo -u postgres initdb --locale=en_US.UTF-8 --encoding=UTF8 -D /var/lib/postgres/data --data-checksums
    ```
1.  Run timescaledb-tune to adjust your `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
    ```bash
    timescaledb-tune
    ```
1. Enable and start the service:
    ```
    systemctl enable postgresql.service
    systemctl start postgresql.service
    ```

> 1.  Restart the service:
>     ```bash
>     systemctl restart postgresql
>     ```

Reviewer: This step is redundant. You started the database in previous step and did no changes that require restart
```suggestion
```

    
</procedure>

## Set up the TimescaleDB extension
When you have PostgreSQL and TimescaleDB installed, you can connect to it from
your local system using the `psql` command-line utility. This is the same tool
you might have used to connect to PostgreSQL before, but if you haven't
installed it yet, check out the [installing psql][install-psql] section.

<procedure>

### Setting up the TimescaleDB extension
1.  On your local system, at the command prompt, connect to the PostgreSQL
    instance as the `postgres` superuser:
    ```bash

>     su postgres -c psql

Reviewer: ```suggestion
    sudo -u postgres psql
```

# Install self-hosted TimescaleDB on archlinux-based systems

> You can host TimescaleDB yourself, on your archlinux system. These

Reviewer 2: ```suggestion
You can host TimescaleDB yourself, on your Arch Linux system. These
```

instructions use the `pacman` package manager.

<highlight type="warning">
If you have already installed PostgreSQL using a method other than the `pacman`
package manager, you could encounter errors following these instructions. It is
safest to remove any existing PostgreSQL installations before you begin. If you
want to keep your current PostgreSQL installation, do not install TimescaleDB
using this method.
[Install from source](/install/latest/self-hosted/installation-source/)
instead.
</highlight>

<procedure>

### Installing self-hosted TimescaleDB on archlinux-based systems
1.  At the command prompt, as root, add the PostgreSQL third party repository
    to get the latest PostgreSQL packages:
    ```bash
    pacman -Syu timescaledb
    ```

> 1.  To use TimescaleDB as PostgreSQL extension, update the `postgresql.conf` file:

Reviewer 2: ```suggestion
1.  Update the `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
```

> 1.  At the command prompt, as root, add the PostgreSQL third party repository
>     to get the latest PostgreSQL packages:
>     ```bash
>     pacman -Syu timescaledb
>     ```
> 1.  Update the `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
>     ```bash
>     echo "shared_preload_libraries = 'timescaledb'" >> /var/lib/postgres/data/postgresql.conf
>     ```
> 1.  Install `timescaledb-tune`:
>     ```bash
>     pacman -Syu timescaledb-tune
>     ```

Reviewer: ```suggestion
1.  Install TimescaleDB and timescaledb-tune:
    ```bash
    pacman -Syu timescaledb timescaledb-tune
    ```
1.  Run timescaledb-tune to adjust your `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
    ```bash
    timescaledb-tune
    ```
```
Since timescaledb is part of the official archlinux repo we do not need to add 3rd party repo.
Instead of manually editing postgresql.conf we should run timescaledb-tune here.

> 1.  Connect to the PostgreSQL instance as the `postgres` superuser:

Reviewer 2: ```suggestion
1.  As the `postgres` user, connect to the PostgreSQL instance:
```

    ```bash
    su -l postgres

>     ```

Reviewer: This command does not connect to the postgres instance but switches to the postgres user

> 1.  As the `postgres` user, connect to the PostgreSQL instance:
>     ```bash
>     su -l postgres
>     ```

Reviewer: ```suggestion
```

> 1.  Initialize the database:

Reviewer: ```suggestion
1.  Initialize the database as the postgres user: 
```

>     ```bash
>     initdb --locale=en_US.UTF-8 --encoding=UTF8 -D /var/lib/postgres/data --data-checksums
>     ```

Reviewer: ```suggestion
    ```bash
    sudo -u postgres initdb --locale=en_US.UTF-8 --encoding=UTF8 -D /var/lib/postgres/data --data-checksums
    ```
```

> 1. Logout as `postgres` superuser:

Reviewer 2: ```suggestion
1. Log out the `postgres` user:
```

> 1. Log out the `postgres` user:
>    ```bash
>    exit
>    ```    

Reviewer: ```suggestion
```

1. Enable and start the service:
    ```
    systemctl enable postgresql.service
    systemctl start postgresql.service

>     ```

Reviewer 2: As you have two commands here, it might be sensible to make this two steps

> 1.  Restart the service after enabling TimescaleDB with `timescaledb-tune`:

Reviewer 2: ```suggestion
1.  Restart the service:
```

    ```bash
    systemctl restart postgresql
    ```
    
</procedure>

> When you have completed the installation, you need to configure your database so
> that you can use it. The easiest way to do this is to run the `timescaledb-tune`
> script, which is included with the `timescaledb-tools` package. For more
> information, see the [configuration][config] section.

Reviewer: ```suggestion
```
We ran timescaledb-tune as part of the installation so this block doesnt apply also the package name is different on archlinux and would be timescaledb-tune

> information, see the [configuration][config] section.

Reviewer 2: It doesn't need to be done in this PR, but we should snippetize this. In fact, I think there's a bunch of stuff in all our install docs that could be snipppetized, so perhaps it's a good idea for someone to go through them all in an organised way.

Author: > It doesn't need to be done in this PR, but we should snippetize this. In fact, I think there's a bunch of stuff in all our install docs that could be snipppetized, so perhaps it's a good idea for someone to go through them all in an organised way.

Yes!!! we do have the create extension and connect to the database in the install guides.

## Set up the TimescaleDB extension
When you have PostgreSQL and TimescaleDB installed, you can connect to it from
your local system using the `psql` command-line utility. This is the same tool
you might have used to connect to PostgreSQL before, but if you haven't

> installed it yet, check out our [installing psql][install-psql] section.

Reviewer 2: ```suggestion
installed it yet, check out the [installing psql][install-psql] section.
```

<procedure>

### Setting up the TimescaleDB extension
1.  On your local system, at the command prompt, connect to the PostgreSQL
    instance as the `postgres` superuser:
    ```bash

>     su postgres -c psql

Reviewer: any reason you are using su instead of sudo here i think sudo is more commonly used for these things

Reviewer 2: We wouldn't usually include sudo at all in commands, but instead say "As root, run the xxx command" (or whatever user). Can we re-edit?

    ```
    If your connection is successful, you'll see a message like this, followed
    by the `psql` prompt:
    ```
    psql (14.3)
    Type "help" for help.
    ```

> 1.  At the `psql` prompt, create an empty database. Our database is

Reviewer 2: ```suggestion
1.  At the `psql` prompt, create an empty database. This database is
```

    called `tsdb`:
    ```sql
    CREATE database tsdb;
    ```
1.  Connect to the database you created:
    ```sql
    \c tsdb
    ```
1.  Add the TimescaleDB extension:
    ```sql
    CREATE EXTENSION IF NOT EXISTS timescaledb;
    ```

> 1.  You can now connect to your database using this command:
>     ```bash
>     su postgres -c 'psql -d tsdb'
>     ```

Reviewer: ```suggestion
1.  You can now connect to your database using this command:
    ```bash
    sudo -u postgres psql tsdb
    ```
```

</procedure>

You can check that the TimescaleDB extension is installed by using the `\dx`
command at the `psql` prompt. It looks like this:
```sql
tsdb=# \dx
                                      List of installed extensions
    Name     | Version |   Schema   |                            Description                            
-------------+---------+------------+-------------------------------------------------------------------
 plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language
 timescaledb | 2.7.0   | public     | Enables scalable inserts and complex queries for time-series data
(2 rows)

```

## Where to next
Now that you have your first TimescaleDB database up and running, you can check

> out the [TimescaleDB][tsdb-docs] section in our documentation, and find out what

Reviewer 2: ```suggestion
out the [TimescaleDB][tsdb-docs] section in the documentation, and find out what
```

you can do with it.

If you want to work through some tutorials to help you get up and running with

> TimescaleDB and time-series data, check out our [tutorials][tutorials] section.

Reviewer 2: ```suggestion
TimescaleDB and time-series data, check out the [tutorials][tutorials] section.
```
